.travis.yml

```diff
@@ -177,7 +177,7 @@ matrix:
       before_script:
         - export CC=arm-linux-gnueabi-gcc CXX=arm-linux-gnueabi-g++
         - mkdir build && cd build
-        - cmake .. -DBUILD_OPENSSL=TRUE -DBUILD_OPENSSL_PLATFORM=linux-generic32 -DBUILD_LIBSRTP_HOST_PLATFORM=x86_64-unknown-linux-gnu -DBUILD_LIBSRTP_DESTINATION_PLATFORM=arm-unknown-linux-uclibcgnueabi
+        - cmake .. -DGENERATE_LIBSRTP_PC=OFF -DBUILD_OPENSSL=TRUE -DBUILD_OPENSSL_PLATFORM=linux-generic32 -DBUILD_LIBSRTP_HOST_PLATFORM=x86_64-unknown-linux-gnu -DBUILD_LIBSRTP_DESTINATION_PLATFORM=arm-unknown-linux-uclibcgnueabi
```

Comment:
Not sure if we want to add yet another flag to our CMake for this use case. Should we infer from `BUILD_LIBSRTP_HOST_PLATFORM` != `BUILD_LIBSRTP_DESTINATION_PLATFORM`. I think this means the same thing, what do you think?

Reply:
Oh! That is a good idea actually!
I have made the change. Lets see if the CI passes.

Reply:
It worked!